h2o-algos/src/main/java/hex/aggregator/Aggregator.java

```diff
@@ -481,4 +484,15 @@ public void map(Chunk c) {
 
     }
   }
+
+  public Model.Parameters.CategoricalEncodingScheme[] supportedCategoricalEncodingSchemes() {
```

Comment:
I think instead of managing a white-list we should black-list the encodings that are not supported (and perhaps add a reason why).

Reply:
Do we want to maintain a blacklist for the possiblity to actually **display** the reason which given encoding is not allowed?

From developer's POV, it makes more sense to me to actually define a set of allowed encodings, rather than subtracting from a global set. Such an approach has one big vulnerability - whenever a new encoding is added, it has to be blacklisted in every algorithm it's not supported at and could be potentially more error prone then explicitly enabling it. The developer usually known the algorithm the encoding is built for, but needn't realize all the places to exclude it from.

From engineering POV, whitelist makes more sense to me.